l10n_ch_base_bank/models/invoice.py

```diff
@@ -39,12 +39,12 @@ def _search(self, args, offset=0, limit=None, order=None, count=False,
                     operator = operator[1:]
                 else:
                     value = '%%%s%%' % (value,)
```

Comment:
Change of indent must be done on the check `if not value:`

You can remove 4 space on line 31 til 41

And leave 42 to 47 to its former indent.